asdf/asdf.py

```diff
@@ -807,9 +807,9 @@ def _parse_header_line(cls, line):
 
         try:
             version = versioning.AsdfVersion(parts[1].decode("ascii"))
-        except ValueError as err:
+        except ValueError:
             msg = f"Unparsable version in ASDF file: {parts[1]}"
-            raise ValueError(msg) from err
+            raise ValueError(msg) from None
```

Comment:
Why has this been changed to `from None`?

Reply:
I reverted this change.